catala.opam

```diff
@@ -75,3 +75,9 @@ depexts: [
   ["py3-pip" "py3-pygments"] {cataladevmode & os-distribution = "alpine"}
   ["python-pygments"] {cataladevmode & os-family = "arch"}
 ]
+pin-depends: [
+  [
+    "dates_calc.0.0.6"
+    "git+https://github.com/CatalaLang/dates-calc#add-c-implem"
```

Comment:
Add a fixme here, so we don't forget when https://github.com/CatalaLang/dates-calc/pull/3 is merged